geochem_inverse_optimize.py

```diff
@@ -143,16 +140,14 @@ def process_element(
 ) -> ElementData:
   # Make a deep copy to avoid over-writing the original data
   reset_sample_network(sample_network)
-
   primary_terms = get_primary_terms(sample_network=sample_network, obs_data=obs_data)
 
   regularizer_terms = get_regularizer_terms(sample_network=sample_network, adjacency_graph=sample_adjacency)
   if not regularizer_terms:
     print("WARNING: No regularizer terms found!")
 
   # Build the objective and constraints
-  # TODO(alexlipp,r-barnes): Should this be a cp.norm(cp.vstack(primary_terms)) to get squaring or should the square happen in the log or should there be no square?
-  objective = cp.sum(primary_terms)  
+  objective = cp.norm(primary_terms)  
```

Comment:
Exactly what I intended. Chucked it in as quick change without checking.
 
Just moving to the 2-norm as to me seems most intuitive norm for what we're doing. i.e., I don't necessarily think we want to encourage sparsity so just didn't really see the need to use the 1-norm instead of the 2-norm which is achieved through norm. Plus aligns with the regularisation term (although I accept that this isn't actually implemented yet...).